build/infrastructure/env/Development/sbq-messagehub-dataavailable.tf

```
@@ -0,0 +1,20 @@
+# Copyright 2020 Energinet DataHub A/S
+#
+# Licensed under the Apache License, Version 2.0 (the "License2");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+module "sbq_messagehub_dataavailable" {
+  source              = "git::https://github.com/Energinet-DataHub/geh-terraform-modules.git//service-bus-queue?ref=2.0.0"
+  name                = "sbq-dataavailable"
```

Suggest to do like this to align with the BundleRequest and BundleReply queues:
```suggestion
  name                = local.MESSAGEHUB_DATAAVAILABLE_QUEUE
```